charts/streams-app/templates/deployment.yaml

```diff
@@ -30,14 +30,27 @@ spec:
     spec:
       affinity:
         podAntiAffinity:
+        {{- if .Values.antiAffinityRequiredEnabled }}
```

Comment:
I don't like this variable name... Also what is the default value?

Reply:
Renamed the variable. The default value is `false`, so the 'preferred anti affinity' is used.